crates/driver/src/domain/competition/auction.rs

```diff
@@ -413,8 +416,11 @@ impl Price {
     }
 
     /// Apply this price to some token amount, converting that token into ETH.
+    ///
+    /// # Example
+    /// 1e18 * 2e18 / 1e18 = 2e18
     pub fn apply(self, amount: eth::TokenAmount) -> eth::Ether {
```

Comment:
Can we call this method `in_eth` to make it extra clear (on top of the return type) what "applying" a price to a token amount means?

Reply:
fixed